test/collateral/valid-with-hashbang/expected-content/test/asyncWithLicense_default.js

```diff
@@ -0,0 +1,50 @@
+#!whatever
```

Comment:
A little variation would be good, if only to prove that we haven't hard-coded the value "whatever" into the parser. Though there may be some slightly more interesting edge cases than, say, "whatever2". "  leading space" and "trailing space  " come to mind.

Reply:
Found more interesting edge cases: the other LineTerminators